Client/Application/ImageIdentifiers.swift

```diff
@@ -55,7 +55,10 @@ public struct ImageIdentifiers {
     public static let nightMode = "menu-NightMode"
     public static let noImageMode = "menu-NoImageMode"
     public static let onboardingWelcome = "tour-Welcome"
+    public static let onboardingWelcomev106 = "onboardingWelcome"
     public static let onboardingSync = "tour-sync-v2"
```

Comment:
Will we be able to remove the old images at one point?

Reply:
yes I can create a ticket to remove for the next release for the one only use in old onboarding